test/manifest_test.py

```diff
@@ -605,3 +605,7 @@ def test_cover_type_on_store(self):
         self.protocol.uncover(test2)
         seal_on_store(self.protocol)
         assert (self.protocol.instructions[-1].lid == "low_evaporation")
+
+    def test_get_protocol_preview(self):
+        preview = get_protocol_preview(self.protocol, "TestMethod", manifest="test/manifest_test.json")
+        assert isinstance(preview, dict)
```

Comment:
Could you match it against the known output instead of just a type check? Thanks!